jitify2.hpp

```diff
@@ -100,16 +103,19 @@
 #endif
 
 #include <algorithm>
```

Comment:
```suggestion
#include <algorithm>
#include <array>
```

Incomplete type `jitify2.hpp:6549` (`std::array<value_type, Size> data_ = {};`) under visual studio 2022.
```